src/js/Color.js

```diff
@@ -0,0 +1,537 @@
+// Copyright 2015 Google Inc.
```

Comment:
I suggest reviewing the two changes in this PR separately. The first one just pulls the code into its own file; the second actually makes modifications.